docs/docs-ref-autogen/office_release/office/office.auth.yml

```diff
@@ -24,6 +24,10 @@ methods:
       -->. This API requires a single sign-on configuration that bridges the add-in to an Azure application. Office
       users sign in with Organizational Accounts and Microsoft Accounts. Microsoft Azure returns tokens intended for
       both user account types to access resources in the Microsoft Graph.
+
+
+      **Important**: In Outlook, this API isn't supported if the add-in is loaded in an Outlook.com or Gmail mailbox.if
+      the add-in is loaded in an Outlook.com or Gmail mailbox.
```

Comment:
why is there an extra clause here (and other instances? this wasn't in DT

Reply:
@davidchesnut sorry, looks like it is in DT but I missed it. Can you fix up and let me know? I'll can help get it reviewed and merged?

Reply:
I went ahead and fixed in PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/62477. Can you re-run GenerateDocs?